### ISSUE TITLE
MAI robots don't gain abilities with research

### ISSUE THREAD

**Issue by OOZ662** [simon-82/ModVarietyPack#111]
v1.24, no other mods installed. No matter which AI research I complete, newly made MAI robots don't gain the related skills, i.e. cooking, crafting, researching, etc. They remain incapable of these tasks.

**Comment by OOZ662** [comment]
Turns out completing all AI research unlocks all abilities at once. I don't know if this is how the base mod behaves, but it's unexpected when there are multiple single research items.

**Comment by simon-82** [comment]
I just tried this. My MAIs gain the skills I researched just fine. It matters that you finish the research first, then activate it. Only then it will have the skills you researched. Look here. For this one I researched cooking and construction. Only these are active, the others are disabled.
![image](https://cloud.githubusercontent.com/assets/14851041/12536660/e58e85ba-c2ab-11e5-8ede-eabdb7234d31.png)
